app/controllers/autocomplete_controller.rb

```diff
@@ -0,0 +1,33 @@
+class AutocompleteController < ApplicationController
+  def github_repos
+    @repos = search_github_repos(params[:query])
+
+    respond_to do |format|
+      format.html { render partial: 'autocomplete/repository_suggestions', locals: { repos: @repos } }
+    end
+  end
+
+  private
+
+  def required_scopes
```

Comment:
Sorry for misunderstanding the meaning of the method. I'll pushed a fix later tonight. 

Thank you for pointing this out.